packages/eslint-config/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@loopback/eslint-config",
+  "version": "1.0.0-1",
+  "description": "Shared configuration for ESLint",
+  "engines": {
+    "node": ">=8.9"
+  },
+  "main": ".eslintrc.js",
+  "author": "IBM Corp.",
+  "copyright.owner": "IBM Corp.",
+  "license": "MIT",
+  "peerDependencies": {
```

Comment:
+1 to use peer dependencies here, it's a good solution until `eslint` implements a better plugin/extension loader.